Find a functionally equivalent code:android.graphics.Paint.setColorFilter:COMMENT
Method Modifier: public      
Comment:/**
 * Set or clear the paint's colorfilter, returning the parameter.
 *
 * @param filter May be null. The new filter to be installed in the paint
 * @return       filter
 */

@@ -1,5 +1,5 @@
 {
-    int filterNative = 0;
+    long filterNative = 0;
     if (filter != null)
         filterNative = filter.native_instance;
     native_setColorFilter(mNativePaint, filterNative);

